### DISCUSSION TITLE
Save state in emulator js

### DISCUSSION THREAD

**Discussion by DaBeybladeMaster** [EmulatorJS/EmulatorJS#777] [General]
I recently have had trouble with creating a save state in emulator js. It worked before but now randomly it would stop the game and not save the game. That happened for about a day until a few days ago when it stopped saving completely. Is there a way to fix this?

**Comment by ethanaobrien** [comment]
Are you using emulatorjs.org or emulatorjs.com

Try at https://demo.emulatorjs.org

**Reply by DaBeybladeMaster** [reply]
.org

**Reply by ethanaobrien** [reply]
Can I have logs?
